packages/salesforcedx-apex-debugger/src/adapter/apexDebug.ts

```diff
@@ -90,13 +91,17 @@ const TRACE_CATEGORY_VARIABLES = 'variables';
 const TRACE_CATEGORY_LAUNCH = 'launch';
 const TRACE_CATEGORY_PROTOCOL = 'protocol';
 const TRACE_CATEGORY_BREAKPOINTS = 'breakpoints';
+const TRACE_CATEGORY_STREAMINGAPI = 'streaming';
```

Comment:
You'll need to update the help text for the trace category: https://github.com/forcedotcom/salesforcedx-vscode/blob/develop/packages/salesforcedx-vscode-apex-debugger/package.nls.json#L14